addons/analytic/models/analytic_account.py

```diff
@@ -115,16 +115,15 @@ def _check_company_consistency(self):
         if self._cr.fetchone():
             raise UserError(_("You can't set a different company on your analytic account since there are some analytic items linked to it."))
 
-    def name_get(self):
-        res = []
+    @api.depends('code', 'partner_id')
```

Comment:
shouldn't it be 
```suggestion
    @api.depends('code', 'partner_id.commercial_partner_id')
```

Reply:
As explained with my bad English in the commit message. I only add the first level  of dependency to avoid too much performance degradation of `modified` method.